installation/scripts/install-tiller.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "The install-tiller.sh script is deprecated and will be removed. Use Kyma CLI instead."
```

Comment:
I wonder if we should delete those warnings, in our case Kyma CLI for this is not an option yet.